src/Stratis.Bitcoin.Features.Wallet/WalletSyncManager.cs

```diff
@@ -95,8 +98,13 @@ public void Stop()
             this.logger.LogTrace("(-)");
         }
 
-        /// <inheritdoc />
-        public virtual void ProcessBlock(Block block)
+        /// <summary>Called when a block is added to the blocksQueue.
```

Comment:
Use `<see>` when you talk about an existing object like the block queue or wallet tip below.